bin/makisu/cmd/build.go

```diff
@@ -216,6 +218,12 @@ func (cmd *buildCmd) newBuildPlan(
 func (cmd *buildCmd) Build(contextDir string) error {
```

Comment:
warning: cyclomatic complexity 20 of function (*buildCmd).Build() is high (> 15) (gocyclo)